core/src/test/java/com/novoda/merlin/service/ResponseCodeValidatorTests.java

```diff
@@ -0,0 +1,88 @@
+package com.novoda.merlin.service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static com.novoda.merlin.service.ResponseCodeValidator.CustomEndpointResponseCodeValidator;
+import static com.novoda.merlin.service.ResponseCodeValidator.DefaultEndpointResponseCodeValidator;
+import static org.fest.assertions.api.Assertions.assertThat;
+
+@RunWith(Enclosed.class)
+public class ResponseCodeValidatorTests {
+
+    private static final int NO_CONTENT = 204;
+
+    @RunWith(Parameterized.class)
+    public static class DefaultEndpointResponseCodeValidatorTest {
+
+        private final int responseCode;
+
+        @Parameterized.Parameters(name = "{0}")
```

Comment:
Usually you have a `[][]` parameteres where you map the `responseCode` to the expected `value`. If you do that but you want to maintain clarity on the output you can also rewrite this `name = "{0}"` to: `name = " response code {0} should return {1}"`

This will allow you to have just one single test! with two tests in place you are running 2xN times, where N is the number of response codes. 

Not a blocker, but could be nicer 😄